src/components/EmailSubscription/EmailSubscription.scss

```diff
@@ -1,3 +1,40 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/responsiveness";
+@use "~stylesheets/typography";
 
-/* Delete these lines when you begin development */
+.EmailSubscription {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 100%;
+
+  .SearchBox {
+    font-family: "Montserrat Italic";
```

Comment:
I know we have designated fonts in `_fonts.scss` and that we were talking to Will about this, but what's the right way to import fonts from that file?

Reply:
That works! Take a look at how it's done in the `_fonts.scss` file. We defined a constant at the top and use that constant throughout the file.